guides/working-with-examples.md

```diff
@@ -0,0 +1,47 @@
+# Working with Examples
+
+Adding meaningful examples can greatly enrich the documentation of components in the Cashmere library. Here are some things to remember when creating or modifying examples.
+
+## Examples files
+
+The following files are used in the creation of examples for the Cashmere documentation website.
+
+### Files you may need to modify
+
+-   `projects/cashmere-examples/src/lib/{EXAMPLE_NAME}` - the folder containing all files for an example
+-   `src/app/core/document-items.service.ts` - the file that adds the example to the appropriate component page in the docs site
```

Comment:
This sentence is a little hard to read. perhaps:
> Service listing all the example components in the docs site

Reply:
noted